src/core/importers/openApi.ts

```diff
@@ -30,13 +30,17 @@ const importSpecs = (
 
   return new Promise((resolve, reject) => {
     if (!schema.openapi || !schema.openapi.startsWith('3.0')) {
-      swagger2openapi.convertObj(schema, {}, (err, { openapi }) => {
-        if (err) {
-          reject(err);
-        } else {
-          resolve(openapi);
-        }
-      });
+      swagger2openapi.convertObj(
+        schema,
+        { patch: true },
```

Comment:
I saw in the doc that you can use that to fix minor errors. Do you have a use case?

Reply:
> I saw in the doc that you can use that to fix minor errors. Do you have a use case?

patch: true fixes my swagger json problems without any side effect. So it may apply to other users.
Or we can add it as an option to the orval configuration.

Reply:
yes maybe, it's better to add it as an option

Reply:
What do you think about adding a 'fix' property who take a boolean in the input config and its default true

Reply:
> What do you think about adding a 'fix' property who take a boolean in the input config and its default true

I think that we must ensure that all swagger2openapi options can be passed with  `patch: true, warnOnly:true` by default
For example: 
```js
swagger2openapi.convertObj(schema,  {patch: true, warnOnly:true, ...userSwagger2openapiOptions}....
```
Note: Default value of patch is true used in here
https://github.com/Mermade/oas-kit/blob/master/packages/swagger2openapi/oas-validate.js#L89

Reply:
seems good to me who would name the property in the orval config?

Reply:
I'm not good at naming. It would be ` swagger2openapi: {..some...options..}`

Reply:
converterOptions: {..some...options..} would be could for you?

Reply:
> converterOptions: {..some...options..} would be could for you?

Yes good.

Reply:
You feel comfortable to add it or I do it?

Reply:
We seem to agree on the job. I can do it.

Reply:
ok top! thanks 🎉